toffy/json_utils_test.py

```diff
@@ -77,3 +77,28 @@ def test_list_moly_fovs(tmpdir):
     pred_moly_fovs = json_utils.list_moly_fovs(tmpdir)
 
     assert np.array_equal(pred_moly_fovs.sort(), moly_fovs.sort())
+
+
+def test_read_json_file():
+    cwd = os.getgwd()
+    # create fake jsons
+    moly_json = {'name': 'bob',
+                 'standardTarget': 'Molybdenum Foil'}
+    json_path = cwd+"/test.json",
+    
+    # write test json
+    with open(json_path, 'w') as jp:
+        json.dump(moly_json, jp)
+
+    # Make sure errors come up when the directory is bad
+     with pytest.raises(ValueError, match='files are the same length'):
+            json_utils.read_json_file("/neasdf1246ljea/asdfje12ua3421ndsf/asdf.json")
+
+    ## Now read json with new function assuming path is legit
+    newfile_test = json_utils.read_json_file(json_path)
+    with open(path, 'r') as jp:
```

Comment:
instead of reading in `newfile_standard`, you should assert that `newfile_test` is the same as `moly_json`